### PR TITLE
complete level 2

### PR DESCRIPTION
**Pull request by DmitryGhoncharko**

### CODE REVIEW

**Review by AntonNovosad — APPROVED** [review]

File: src/main/java/task1/Run.java [hunk 1]
```diff
@@ -17,5 +17,54 @@ public static void main(String[] args) {
         }
         System.out.println(max);
         System.out.println(midTaxCoefficient / vehicleTypes.length);
+
+        Vehicle[] cars = {new Vehicle(vehicleTypes[0], "Volkswagen Crafter", "5427 AX-7", 2022, 2015, 37600, Color.BLUE, 100.2),
```
Comment (AlexeyKatomachin) [inline src/main/java/task1/Run.java:21]: separate to init method

File: src/main/java/task1/Vehicle.java [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    }
+
+    public double getTankCapacity() {
+        return tankCapacity;
+    }
+
+    public void setTankCapacity(double tankCapacity) {
+        this.tankCapacity = tankCapacity;
+    }
+
```
Comment (AlexeyKatomachin) [inline src/main/java/task1/Vehicle.java:143]: remove empty row

File: src/main/java/task1/Run.java [hunk 3]
```diff
@@ -17,5 +17,54 @@ public static void main(String[] args) {
...
+    }
+
+    private static class VehicleUtils {
+        private static void printAllVehicles(Vehicle[] vehicles) {
+            for (Vehicle vehicle : vehicles) {
+                System.out.println(vehicle.toString());
+            }
+        }
+    }
+
```
Comment (AlexeyKatomachin) [inline src/main/java/task1/Run.java:69]: remove empty row

File: src/main/java/task1/Run.java (hunk 1 above)
Comment (DmitryGhoncharko) [reply src/main/java/task1/Run.java:32]: fixed

File: src/main/java/task1/Vehicle.java (hunk 2 above)
Comment (DmitryGhoncharko) [reply src/main/java/task1/Vehicle.java:143]: fixed

File: src/main/java/task1/Run.java (hunk 3 above)
Comment (DmitryGhoncharko) [reply src/main/java/task1/Run.java:78]: fixed

**Review by AntonNovosad — APPROVED** [review]

**Pull request merged**